src/services/development.ts

```diff
@@ -5,20 +5,36 @@ import { merge, build, ServiceDefinition, token, get } from './utils'
 import { useBootstrap } from '../index'
 import { mocks, setupHandlers } from '@/api/mocks'
 import { fs, fakeApi } from '@/api/mocks/index'
+import type { FS, FakeEndpoint } from '@/api/mocks/index'
 import CookiedEnv from '@/services/env/CookiedEnv'
 import Logger from '@/services/logger/DatadogLogger'
 import { disabledLogger } from '@/services/logger/DisabledLogger'
 import type { TokenType } from '@/services/utils'
 
 export { constant, get, container, createInjections, build, merge } from './utils'
 
-type Msw = ReturnType<typeof setupWorker>
+type Msw = {
+  listen: () => void
+  resetHandlers: () => void
+  close: () => void
+} & ReturnType<typeof setupWorker>
+
 const $ = {
   ...PROD_TOKENS,
   msw: token<Msw>('msw'),
   mswKumaHandlers: token<RestHandler[]>('msw.kuma.handlers'),
   mswKumaMocks: token<RestHandler[]>('msw.kuma.mocks'),
+  kumaFS: token<Record<string, FakeEndpoint>>('fake.fs.kuma'),
+  /**
+   * @description
+   * Service Label for labeling MSW handlers for consumption via setupWorker
+   */
   mswHandlers: token<RestHandler[]>('msw.handlers'),
+  /**
+   * @description
+   * Service Label for labeling fake FSs for consumption via MSW
+   */
```

Comment:
Oh I wasn't gonna add these here! Are these helpful or not? I'm hoping they'll help when you need to use these tokens in other files.

Reply:
Yes, those comments will be added to tooltips in editors when hovering over, for example, `TOKENS.api`. Though personally I would not add `@description` because it doesn’t really add any value. The main comment content will be added, too, just with the difference that it won’t be prefixed with `@description`.